.travis.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  - docker
+
+script:
+  # Script for Building Docker Image and Run the Tests on TRAVIS CI
+  - docker build --target test --tag testing_modules .
+  - docker run testing_modules test_ut_it_modules
+ # - docker run --env API_KEY --env API_TOKEN --env NAME --env BOARD_ID --env TODO_LIST_ID --env DOING_LIST_ID --env DONE_LIST_ID --env FLASK_APP --env FLASK_ENV testing_modules test_e2e_module
```

Comment:
Why is the e2e test run commented out - were you facing issues?

Note that you shouldn't need to pass all these variables in:
- The e2e should create its own temporary board, so you **shouldn't** need to pass in BOARD_ID, TODO_LIST_ID, DOING_LIST_ID, DONE_LIST_ID.
- And FLASK_APP and FLASK_ENV shouldn't be necessary for running the app. Even if they were, they're the kind of thing you could set in the Dockerfile.